CommandLine/testing/GitHub-ImageDiff.sh

```diff
@@ -57,55 +57,65 @@ if [[ ! -z "${com_master_pr}" ]]; then
 
   echo -e "${deleted_images_comment}"
   echo "Aborting!"
-  enigmabot_post_comment "${deleted_images_comment}"
-  travis_terminate 1
+  if [[ "$TRAVIS" -eq "true" ]]; then
+    enigmabot_post_comment "${deleted_images_comment}"
```

Comment:
This makes me happy. Well done.

Reply:
Good then, it makes fundies happy too because he can run it locally, and that's a good thing. Arguably, this whole thing is easier to run locally than it was before.